src/main/java/seedu/address/model/EntityList/IssueList.java

```diff
@@ -87,7 +85,7 @@ public void delete(Id id) throws Exception {
      * @return boolean
      */
     @Override
-    public boolean isContain(Id id) {
+    public boolean contains(Id id) {
         for (Issue i: this.issues) {
             if (i.getId() == id) {
```

Comment:
If we're going to have multiple copies for same Entities, don't we need to use equals()?

Reply:
Thanks for the catch, yes we should, especially since the equals method has already been implemented.

Reply:
Yep will change all to equals, forgot that I implemented equals for everything lol 😆 . thanks for the catch